Sources/Subprocess/Configuration.swift

```diff
@@ -516,6 +516,40 @@ extension Executable: CustomStringConvertible, CustomDebugStringConvertible {
     }
 }
 
+extension Executable {
+    internal func possibleExecutablePaths(
+        withPathValue pathValue: String?
+    ) -> Set<String> {
+        switch self.storage {
+        case .executable(let executableName):
+            #if os(Windows)
+            // Windows CreateProcessW accepts executable name directly
+            return Set([executableName])
```

Comment:
Even if CreateProcessW accepts an executable name directly, isn't it bad that we're ignoring the passed-in path value? I know this is internal but might not match the expected semantics of the caller.